modyn/tests/storage/internal/file_wrapper/binary_file_wrapper_test.cpp

```diff
@@ -24,30 +25,70 @@ class BinaryFileWrapperTest : public ::testing::Test {
       : config_{StorageTestUtils::get_dummy_file_wrapper_config()},
         filesystem_wrapper_{std::make_shared<MockFilesystemWrapper>()} {
     file_name_ = tmp_dir_ + "/test.bin";
+    file_name_endian_ = tmp_dir_ + "/test_endian.bin";
   }
 
   void SetUp() override {
     std::filesystem::create_directory(tmp_dir_);
 
     std::ofstream file(file_name_, std::ios::binary);
-    const std::vector<std::pair<uint32_t, uint16_t>> data = {{42, 12}, {43, 13}, {44, 14}, {45, 15}};
+    const std::vector<std::pair<uint16_t, uint16_t>> data = {{42, 12}, {43, 13}, {44, 14}, {45, 15}};
```

Comment:
Why this change?

Reply:
The only place we a using those values is in the `payload_to_file` function. It takes `uint16_t` inputs. As we don't need 4 bytes for our tests, I went for `uint16_t` over `uint32_t` everywhere.